packages/core/admin/ee/admin/pages/Roles/CreatePage/index.js

```diff
@@ -142,7 +143,18 @@ const CreatePage = () => {
       >
         {({ handleSubmit, values, errors, handleReset, handleChange, handleBlur }) => (
           <form onSubmit={handleSubmit}>
-            <ContainerFluid padding="0">
+            <>
+              <HeaderLayout
+                primaryAction={<Button startIcon={<AddIcon />}>Add an entry</Button>}
+                secondaryAction={(
+                  <Button variant="tertiary" startIcon={<EditIcon />}>
+                    Edit
+                  </Button>
+                )}
+                title="Other CT"
+                subtitle="36 entries found"
+                as="h2"
```

Comment:
```suggestion
                as="h1"
```

is appropriate on this one 😊

Reply:
OH this was not supposed to be there, this is the wip for the new branch, but anyways ^^'